onchain/src/art_peace.cairo

```diff
@@ -971,7 +1006,20 @@ pub mod ArtPeace {
             // TODO: Allow some threshold?
             if matches == template_metadata.width * template_metadata.height {
                 self.templates.completed_templates.write(template_id, true);
-            // TODO: Distribute rewards
+                // Distribute rewards
+                if pixel_contributors.len() > 0 {
+                    let mut i = 0;
+
+                    while i < pixel_contributors
+                        .len() {
+                            let mut user = *pixel_contributors.at(i).into();
```

Comment:
you should be able to define this like `let user: felt252 = *pixel_contributors.at(i).into();` and not have to convert on another line.